src/App.js

```diff
@@ -1,45 +1,68 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import Container from '@material-ui/core/Container';
-
 import './App.css';
+import { AuthContext } from './components/Auth/AuthContext';
 import About from './components/About';
 import Resources from './components/Resources';
 import Home from './components/Home';
 import Nav from './components/Nav';
+import LoginForm from './components/Auth/LoginForm.js';
+import SignUpForm from './components/Auth/SignUpForm.js';
 import Coworking from './components/Coworking';
+import Profile from './components/Profile';
 import SubmitResource from './components/Resources/submitResource';
 import ResourcePage from './components/Resources/ResourcePage.js';
+import PrivateRoute from './PrivateRoute';
 
 function App() {
+  const [authTokens, setAuthTokens] = useState(
+    JSON.parse(localStorage.getItem('tokens')) || undefined
+  );
+  const setTokens = data => {
+    // TODO: Consider using cookies instead of JSON tokens
+    localStorage.setItem('tokens', JSON.stringify(data));
+    setAuthTokens(data);
+  };
+
+  console.log(authTokens);
```

Comment:
Let's not log `authTokens` in production 😅

Reply:
Haha, I actually caught this while recording my fifteen-minute walkthrough at https://gitduck.com/watch/5e813023b7d7b8f33995fc6b! Removed as part of f9d7213.